storage/Devices/MdImpl.cc

```diff
@@ -291,25 +313,136 @@ namespace storage
     }
 
 
+    void
+    Md::Impl::calculate_region_and_topology()
+    {
+	vector<BlkDevice*> devices = get_devices();
+
+	long real_chunk_size = chunk_size;
+
+	if (real_chunk_size == 0)
+	    real_chunk_size = get_default_chunk_size();
+
+	// mdadm uses a chunk size of 64 KiB just in case the RAID1 is ever reshaped to RAID5.
+	if (md_level == RAID1)
+	    real_chunk_size = 64 * KiB;
+
+	int number = 0;
+	unsigned long long sum = 0;
```

Comment:
Since this is used so many times here now, how about

    typedef unsigned long long DiskSize;

?
Not only would that be shorter to write, it would also clearly point out what kind of thing this is.